yb-voyager/cmd/common.go

```diff
@@ -452,3 +452,34 @@ func nameContainsCapitalLetter(name string) bool {
 	}
 	return false
 }
+
+func checkCutoverCompleted() bool {
```

Comment:
Return string representation as reported by the `cutover status` command.
Updated the `cutoverStatus.go:checkAndReportCutoverStatus()` to call this function to GET the status.